datalad/support/sshconnector.py

```diff
@@ -577,12 +577,11 @@ def close(self, allow_fail=True, ctrl_path=None):
         allow_fail: bool, optional
           If True, swallow exceptions which might be thrown during
           connection.close, and just log them at DEBUG level
-        ctrl_path: str or list of str, optional
+        ctrl_path: str, Path, or list of str or Path, optional
           If specified, only the path(s) provided would be considered
         """
         if self._connections:
-            from datalad.utils import assure_list
-            ctrl_paths = assure_list(ctrl_path)
+            ctrl_paths = [Path(p) for p in assure_list(ctrl_path)]
```

Comment:
I guess as with imports, to harmonize ensure vs assure we could start using ensure more in such diffs